src/test/scala/org/broadinstitute/dsde/workbench/sam/dataAccess/MockDirectoryDAO.scala

```diff
@@ -290,4 +290,30 @@ class MockDirectoryDAO(val groups: mutable.Map[WorkbenchGroupIdentity, Workbench
 
   override def loadUserByGoogleSubjectId(userId: GoogleSubjectId, samRequestContext: SamRequestContext): IO[Option[SamUser]] =
     IO.pure(users.values.find(_.googleSubjectId.contains(userId)))
+
+  override def acceptTermsOfService(userId: WorkbenchUserId, tosVersion: String, samRequestContext: SamRequestContext): IO[Boolean] = {
+    loadUser(userId, samRequestContext).map {
+      case None => false
+      case Some(user) =>
+        if (user.acceptedTosVersion.contains(tosVersion)) {
```

Comment:
minor:
```suggestion
        if (user.acceptedTosVersion.equals(tosVersion)) {
```

Reply:
`acceptedTosVersion` is an `Option` so I think `contains` is correct